resources/scripts/generate-build-data.sh

```diff
@@ -417,4 +418,13 @@ fetchAndPrepareBuildInfo "${BUILD_INFO}" "${BO_BUILD_URL}/" "build" "${DEFAULT_H
 prepareEnvInfo "${ENV_INFO}" "env"
 echo '}' >> "${BUILD_REPORT}"
 
+### For each entry in the test map then create a flatten document
+N=0
+jq -c '.test = (.test[])' "${BUILD_REPORT}" |
+while read -r json ; do
+  N=$((N+1))
+  echo "{ \"index\":{} }" >> "${BUILD_BULK_REPORT}"
+  echo "${json}" >> "${BUILD_BULK_REPORT}"
```

Comment:
```suggestion
  echo "{ \"doc\" : ${json} }" >> "${BUILD_BULK_REPORT}"
```

Reply:
Cool, I just added some tests to ensure the `ndjson` format works as expected, and in fact `-c` is the key flag for the jq